sdk/storage/src/blob/container/mod.rs

```diff
@@ -102,10 +111,7 @@ impl Container {
         }
     }
 
-    pub(crate) fn from_response(
-        name: String,
-        headers: &HeaderMap,
-    ) -> Result<Container, AzureError> {
+    pub(crate) fn from_response(name: &str, headers: &HeaderMap) -> Result<Container, AzureError> {
```

Comment:
I think it's better to take `String` or at least a `Into<String>` as this makes it clearer that you need ownership of the string data. The user could have an owned string they're ready to give you, but they're required to pass a slice that you'll copy instead

Reply:
You are absolutely right! 

(side note: this is probably the oldest code left in the crate... and it shows 🤣 )